src/reporter.ts

```diff
@@ -53,17 +54,26 @@ export const defaultReporter = <T = any>(
     RULE,
     invalidVarsOutput.sort().join('\n'),
     missingVarsOutput.sort().join('\n'),
-    colors.yellow('\n Exiting with error code 1'),
     RULE,
   ]
     .filter((x) => !!x)
     .join('\n')
 
   logger(output)
+}
+
+export const defaultReporter = <T = any>(
+  { errors = {} }: ReporterOptions<T>,
+  { onError, logger }: ExtraOptions<T> = { logger: defaultLogger },
+) => {
+  if (!Object.keys(errors).length) return
+
+  envalidLogger(errors, logger);
```

Comment:
Because the noun `logger` is already used in this module, I think renaming this to `envalidErrorFormatter` would make things a bit more clear. What do you think?

Reply:
On one hand, it's a logger becaues it does log things, on the other hand, it also does formatting. So I guess it's both?
I do not really care, so if you want it called `envalidErrorFormatter` I'll just make the changes.

Reply:
Sure, let's go with `envalidErrorFormatter`